chalice/app.py

```diff
@@ -418,12 +418,12 @@ class Chalice(object):
 
     FORMAT_STRING = '%(name)s - %(levelname)s - %(message)s'
 
-    def __init__(self, app_name, configure_logs=True, env=None):
+    def __init__(self, app_name, debug=False, configure_logs=True, env=None):
```

Comment:
Do we want to put this argument at the end just in case users are providing these options as positional arguments?